src/main/java/auth/BasicAuth.java

```diff
@@ -29,4 +29,9 @@ public HttpHeaders getAuthHeaders() {
             }
         };
     }
+
+    @Override
+    public String toString(){
+        return "BasicAuth [username=" + username + ", password=" + password + "]";
```

Comment:
also i tend to like this `toString` format best, would be awesome if it was consistent throughout